dask_searchcv/tests/test_adaptive.py

```diff
@@ -0,0 +1,179 @@
+from dask_searchcv.adaptive import _top_k, Hyperband
+from dask_ml.linear_model import PartialSGDClassifier
+from dask_ml.datasets import make_classification
+import numpy as np
+import dask.array as da
+from dask.distributed import Client
+import pandas as pd
+import pytest
+import distributed
+from pprint import pprint
+import scipy.stats as stats
+import random
+from sklearn.linear_model import Lasso
+
+
+class TestFunction:
```

Comment:
Assuming this is just a helper, the test name shouldn't start with `Test` as then it gets discovered by test runners (they typically pick up tests named `Test*` and functions named `test_*`.